web_download_disable_iframe/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+==============================
+Download files in a new window
+==============================
+
+This module disables downloading files via a hidden iframe, which doesn't work
+on some more obscure and/or old browsers. Downloads are triggered in a new
+window, the way it's done for iOS browsers in the standard code.
+
+Usage
+=====
+
+* go to ...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+    :alt: Try me on Runbot
+    :target: https://runbot.odoo-community.org/runbot/162/8.0
+
```

Comment:
Remember that this button is only valid when the PR is merged, not now. For reviewing the change, you should go to the runbot on the CIs part of the PR.